autoload/go/issue.vim

```diff
@@ -0,0 +1,34 @@
+let s:templatepath = go#util#Join(fnamemodify('<sfile>', ':p:h'), '.github', 'ISSUE_TEMPLATE.md')
```

Comment:
This doesn't seem to work:

```
Error detected while processing function go#issue#New[1]..<SNR>104_issuebody:
line    1:
E484: Can't open file /home/martin/work/src/github.com/teamwork/desk/.github/ISSUE_TEMPLATE.md
```

The (only) file I have open is `/home/martin/work/src/github.com/teamwork/desk/main.go`.

Reply:
How did you source the file?

Reply:
hmm, I can duplicate what you're seeing. I'll investigate later.